test/testG2oParser.jl

```diff
@@ -26,13 +26,13 @@ end
 @testset "Test g2o file parsing." begin
 
   reflines = [
-              "EDGE_SE2 0 1 10.0 0.0 1.0471975511965976 10.0 10.0 0.0 0.0 10.0 0.0 0.0 0.0";
+              "EDGE_SE2 0 1 10.0 0.0 1.0471975511965976 10.0 0.0 0.0 10.0 0.0 10.0";
               "LANDMARK 0 2 0.0 20.0 3.162277660168379 0.0 1.0";
-              "EDGE_SE2 1 3 10.0 0.0 1.0471975511965976 10.0 10.0 0.0 0.0 10.0 0.0 0.0 0.0";
-              "EDGE_SE2 3 4 10.0 0.0 1.0471975511965976 10.0 10.0 0.0 0.0 10.0 0.0 0.0 0.0";
-              "EDGE_SE2 4 5 10.0 0.0 1.0471975511965976 10.0 10.0 0.0 0.0 10.0 0.0 0.0 0.0";
-              "EDGE_SE2 5 6 10.0 0.0 1.0471975511965976 10.0 10.0 0.0 0.0 10.0 0.0 0.0 0.0";
-              "EDGE_SE2 6 7 10.0 0.0 1.0471975511965976 10.0 10.0 0.0 0.0 10.0 0.0 0.0 0.0";
+              "EDGE_SE2 1 3 10.0 0.0 1.0471975511965976 10.0 0.0 0.0 10.0 0.0 10.0";
+              "EDGE_SE2 3 4 10.0 0.0 1.0471975511965976 10.0 0.0 0.0 10.0 0.0 10.0";
+              "EDGE_SE2 4 5 10.0 0.0 1.0471975511965976 10.0 0.0 0.0 10.0 0.0 10.0";
+              "EDGE_SE2 5 6 10.0 0.0 1.0471975511965976 10.0 0.0 0.0 10.0 0.0 10.0";
+              "EDGE_SE2 6 7 10.0 0.0 1.0471975511965976 10.0 0.0 0.0 10.0 0.0 10.0";
```

Comment:
@dehann , can you please just confirm that this test is now correct as you wrote the last one?

Reply:
I think its correct: https://github.com/RainerKuemmerle/g2o/wiki/File-Format#edges--constraints